rs/nns/integration_tests/src/cycles_minting_canister.rs

```diff
@@ -4,13 +4,14 @@ use cycles_minting_canister::{
     CanisterSettingsArgs, ChangeSubnetTypeAssignmentArgs, CreateCanister, CreateCanisterError,
     IcpXdrConversionRateCertifiedResponse, NotifyCreateCanister, NotifyError, NotifyMintCyclesArg,
     NotifyMintCyclesSuccess, SubnetListWithType, SubnetTypesToSubnetsResponse,
-    UpdateSubnetTypeArgs, BAD_REQUEST_CYCLES_PENALTY, CYCLES_LEDGER_CANISTER_ID,
-    MEMO_CREATE_CANISTER, MEMO_MINT_CYCLES, MEMO_TOP_UP_CANISTER,
+    UpdateSubnetTypeArgs, BAD_REQUEST_CYCLES_PENALTY, MEMO_CREATE_CANISTER, MEMO_MINT_CYCLES,
+    MEMO_TOP_UP_CANISTER,
 };
 use dfn_candid::candid_one;
 use dfn_protobuf::protobuf;
 use ic_canister_client_sender::Sender;
 use ic_ledger_core::tokens::{CheckedAdd, CheckedSub};
+use ic_nns_constants::{CYCLES_LEDGER_CANISTER_ID, CYCLES_MINTING_CANISTER_ID};
```

Comment:
is CYCLES_MINTING_CANISTER_ID used somewhere?

Reply:
I don't know why I added it... There is actually another `use ic_nns_constants::...` block somewhere under that line which already imports `CYCLES_MINTING_CANISTER_ID`. Merged the 2 statements.